### ISSUE TITLE
Work In Progress: Import golang/protobuf/proto instead of gogo/protobuf/proto

### ISSUE THREAD

**Issue by awalterschulze** [gogo/protobuf#55]
This idea was first mentioned in https://github.com/gogo/protobuf/issues/41
, but I thought I'd rather split it out into a separate issue.

If you would like the generated code to rather import github.com/golang/protobuf/proto instead of github.com/gogo/protobuf/proto please vote here, by commenting below.

My plan is to add another extension so that you can set certain files to rather import golang/protobuf.
This way it is backwards compatible with the current implementation.

I also suspect that these users will be users that don't want to use extensions or import gogo.proto in their proto files.
The solution to this problem will be provided by the solution to https://github.com/gogo/protobuf/issues/39

In other words with a protoc-gen-gogovanity binary you could get all the code generation you want without depending on gogoprotobuf.

**Comment by awalterschulze** [comment]
I have done the work on the nodep branch
Here is the commit
https://github.com/gogo/protobuf/commit/4549dd00f4bec9c39864fdee3484c54bf723555b
If someone would try this out before I merge it back into proto3, it would be greatly appreciated.

**Comment by awalterschulze** [comment]
Merged into proto3
